### DISCUSSION TITLE
Wazuh cluster not showing logs

### DISCUSSION THREAD

**Discussion by EvaSzlifierzCriado** [wazuh/wazuh#16018] [Q&A]
I have configured my firewall in Fortigate to send logs to my Wazuh cluster server. When I try both TCP and UDP protocols in another server (private or public) using Logstash they are send and received properly in port 5144. But when I try in Wazuh cluster (using logstash as well) they arrive if the protocol is TCP but not with UDP. I have configured the wazuh cluster so that it can receive logs in the ip and port that I specify in logstash configuration. Also, I check the port and it's listenning. 
Can someone explain me why is this happening?

**Comment by mdiego92** [comment]
Hello @evaa1234,

Are you fetching the Fortigate logs through rsyslog or logcollector? Could you please share with us your /var/ossec/etc/ossec.conf file to further analyze the issue?

Thank you very much.

**Comment by EvaSzlifierzCriado** [comment]
I don't know why but I can't paste my ossec.conf poperly. I attach you an slack link where I comment the same issue with the ossec.conf in a response. Also, I understand that I use rsyslog since I use syslog and it is a variant, right?

Slack link: https://wazuh.slack.com/archives/C0A933R8E/p1675154150425769

Thank you!

**Reply by mdiego92** [reply]
Hello @evaa1234,

As per your current ossec.conf configuration, you will need to add the TCP protocol to your syslog configuration:

```
  <remote>
    <connection>syslog</connection>
    <port>5144</port>
    <protocol>udp,tcp</protocol>
    <allowed-ips>0.0.0.0/0</allowed-ips>
  </remote>
```

Documentation for reference: https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/remote.html#protocol

Since Wazuh V4.2.0 it is possible to configure both UDP and TCP to work simultaneously, and I have noticed you're using Wazuh V4.3.9.

After performing that modification, you will need to restart the Wazuh Manager.
Please let me know if that configuration worked for you.

**Reply by EvaSzlifierzCriado** [reply]
Yes, I have configured both tcp and udp as you said but the thing is that I want to check if the logs arrive to wazuh. The only way that has occurred to me to do it is to install logstash in a wazuh pod and check if logs arrive from the firewall to my wazuh pod, independently from the ports configured on my ossec. 

That is, I want my logs to arrive at port 515 for example if I have this port open on the pod, independently from the port configured on my ossec at syslog. I want to treat my wazuh pod as a normal server which receives the logs without using the syslog part on ossec, if that makes sense. If this works and I can see the logs, I will configure the ossec as you previously said.

Do you know how can I do that and/or if there is another way to check that the logs arrive?

Thank you very much.
